lua/hfcc/completion.lua

```diff
@@ -142,8 +143,9 @@ function M.create_autocmds()
   api.nvim_create_augroup(augroup, { clear = true })
 
   api.nvim_create_autocmd("InsertLeave", { pattern = "*", callback = M.cancel })
+  api.nvim_create_autocmd("CursorMovedI", { pattern = "*", callback = M.cancel })
 
-  api.nvim_create_autocmd("CursorMovedI", {
+  api.nvim_create_autocmd("CursorHoldI", {
```

Comment:
could this be made configurable? this way user have the choice between tuning `opts.debounce_ms` or enabling `CursorHoldI`.

Reply:
For me debounce doesn't work, I tried increasing it but it didn't change a thing for me.
Didn't have time to screencast the behaviour.

Reply:
I'll try running some tests locally this week, keep you posted

Reply:
Found the culprit :)

cf #29